### PR TITLE
fix: fixes to allergy card in patient summary

### PR DESCRIPTION
**Pull request by bradymiller**
fixes to allergy card in patient summary

### CODE REVIEW

File: interface/patient_file/summary/demographics.php [hunk 1]
```diff
@@ -1058,16 +1058,7 @@ function filterActiveIssues(array $i): array
                     // ALLERGY CARD
                     if ($allergy === 1) {
                         $allergyService = new AllergyIntoleranceService();
-                        $_rawAllergies = filterActiveIssues($allergyService->getAll(['lists.pid' => $pid])->getData());
```
Comment (bradymiller) [inline interface/patient_file/summary/demographics.php:1061]: note that the original `getAll` call was returning nothing

File: src/Common/Layouts/LayoutsUtils.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    public static function getListItemTitle($list, $option)
+    {
+        $row = sqlQuery("SELECT `title` FROM `list_options` WHERE `list_id` = ? AND `option_id` = ? AND activity = 1", [$list, $option]);
+        if (empty($row['title'])) {
+            return $option;
+        }
+        return xl_list_label($row['title']);
+    }
+}
```
Comment (stephenwaite) [inline src/Common/Layouts/LayoutsUtils.php:25]: EtiIRpM0

**Comment by bradymiller** [comment]
This is testing well and good to go. Will bring this in (and change the function call to original GetAll call) after another PR is in the codebase: https://github.com/openemr/openemr/pull/6864

**Comment by stephenwaite** [comment]
southpark_cheering_edited

**Comment by stephenwaite** [comment]
all set for you @bradymiller 

EtiIRpM0

**Pull request merged**